src/main/java/com/box/sdk/BoxDeserializationException.java

```diff
@@ -0,0 +1,50 @@
+package com.box.sdk;
+
+/**
+ * A class representing exceptions caused from deserializing errors.
+ */
+public class BoxDeserializationException extends RuntimeException {
+    private String fieldName;
+    private String fieldValue;
+    private String errorMessage;
+
+    /**
+     * Initializes the BoxDeserializationException class.
+     *
+     * @param member the key of the json member the deserialization occurred on.
+     * @param value the value of the json member the deserialization occurred on.
+     * @param className the name of the class the deserialization occurred on.
+     * @param e the throwable cause for the exception.
+     */
+    public BoxDeserializationException(String member, String value, String className, Exception e) {
```

Comment:
Now that I think about it, the class where the exception was thrown from will be right at the top of the stack trace; we might want to remove the class name from the exception message.  That should clean up all the call sites where these exceptions are being constructed